docs/_docs/clustering/network-configuration.adoc

```diff
@@ -49,10 +49,18 @@ tab:C++[unsupported]
 The following table describes some most important properties of `TcpDiscoverySpi`.
 You can find the complete list of properties in the javadoc:org.apache.ignite.spi.discovery.tcp.TcpDiscoverySpi[] javadoc.
 
+[CAUTION]
+====
+You should assign `IgniteConfiguration.localHost` or `TcpDiscoverySpi.localAddress`.
+By default, node obtains all available IP addresses. Several node addresses can prolong detection of node failure.
+You should not allow node bind to multiple ip addresses until they represent some real physical connections which can
+give you more reliability.
+====
+
 [cols="1,2,1",opts="header"]
 |===
 |Property | Description| Default Value
-| `localAddress`| Local host IP address used for discovery. | By default, the node uses the first non-loopback address it finds. If there is no non-loopback address available, then `java.net.InetAddress.getLocalHost()` is used.
+| `localAddress`| Local host IP address used for discovery. Prevails over `IgniteConfiguration.localHost`. | By default, all available addresses are assigned to node. If there is no non-loopback address available, then `java.net.InetAddress.getLocalHost()` is used.
```

Comment:
Please adjust to the following:

_... If set, overrides the `IgniteConfiguration.localHost` setting. | By default, a node binds to all available network addresses. If there is a non-loopback address available, then `java.net.InetAddress.getLocalHost()` is used._

Reply:
Fixed.